packages/slate-cli/src/commands/migrate.js

```diff
@@ -4,7 +4,7 @@ import {join} from 'path';
 import {prompt} from 'inquirer';
 import {green, red, yellow} from 'chalk';
 import figures from 'figures';
-import {downloadFromUrl, startProcess, writePackageJsonSync, move, isShopifyTheme, isShopifyThemeWhitelistedDir} from '../utils';
+import {downloadFromUrl, startProcess, writePackageJsonSync, move, beautifyJson, isShopifyTheme, isShopifyThemeSettingsFile, isShopifyThemeWhitelistedDir} from '../utils';
```

Comment:
I feel like this declaration is overkill -- what's your opinion on importing the whole `utils` object and call the methods from that?

Reply:
Agreed.

Reply:
Ditto!